src/main/java/WayofTime/bloodmagic/ritual/crushing/CrushingHandlerBasicCuttingFluid.java

```diff
@@ -0,0 +1,42 @@
+package WayofTime.bloodmagic.ritual.crushing;
+
+import WayofTime.bloodmagic.api.impl.BloodMagicAPI;
+import WayofTime.bloodmagic.api.impl.recipe.RecipeAlchemyTable;
+import WayofTime.bloodmagic.item.alchemy.ItemCuttingFluid;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CrushingHandlerBasicCuttingFluid implements ICrushingHandler {
```

Comment:
These 2 implementations could be combined into a single class with constructor parameters.